internal/scanner/scanner.go

```diff
@@ -79,15 +81,15 @@ func New(
 	}
 
 	return &Scanner{
-		logger:           logger,
-		cfg:              cfg,
-		db:               db,
-		httpClient:       httpClient,
-		grpcConn:         grpcConn,
-		requestTemplates: requestTemplates,
-		router:           router,
-		wsClient:         websocket.DefaultDialer,
-		isTestEnv:        isTestEnv,
+		logger:            logger,
+		cfg:               cfg,
+		db:                db,
+		httpClient:        httpClient,
+		grpcConn:          grpcConn,
+		requestTemplates:  requestTemplates,
+		router:            router,
+		wsClient:          websocket.DefaultDialer,
+		enableDebugHeader: addDebugHeader,
```

Comment:
Maybe it is better to use the same one name, `enableDebugHeader` or `addDebugHeader`, because they basically do the same, so there is probably no reason to do 2 flags with the same logic but different names